modules/imgcodecs/src/loadsave.cpp

```diff
@@ -639,6 +639,27 @@ Mat imread( const String& filename, int flags )
     return img;
 }
 
+
+/**
+ * Read an image
+ *
+ *  This function merely calls the actual implementation above and returns itself.
+ *
+ * @param[in] filename File to load
+ * @param[in] img object in which the image will be loaded
+ * @param[in] flags Flags you wish to set.
+*/
+Mat imread( const String& filename, Mat & img, int flags )
```

Comment:
Please try this method with a ROI:
```cpp
Mat src = Mat::zeros(1000, 1000, CV_8UC3);
imread("image_640x480.png", src.rowRange(0, 480).colRange(0, 640));
```
Expected that final image it located in left-top corner and zero (black) pixels are not corrupted.

Adding a test like this also would be nice.